src/keepers/AlchemixHarvester.sol

```diff
@@ -22,15 +23,18 @@ contract AlchemixHarvester is IAlchemixHarvester, AlchemixGelatoKeeper {
     resolver = _resolver;
   }
 
-  /// @notice Runs a the specified harvest job.
+  /// @notice Runs a the specified harvest job and donates optimism rewards.
   ///
-  /// @param alchemist        The address of the target alchemist.
-  /// @param yieldToken       The address of the target yield token.
-  /// @param minimumAmountOut The minimum amount of tokens expected to be harvested.
+  /// @param alchemist                The address of the target alchemist.
+  /// @param yieldToken               The address of the target yield token.
+  /// @param minimumAmountOut         The minimum amount of tokens expected to be harvested.
+  /// @param expectedRewardsExchange  The minimum VSP to debt tokens.
   function harvest(
     address alchemist,
+    address rewardCollector,
```

Comment:
instead of passing this in, lets set a mapping on this contract of {yieldToken => rewardCollector}.  passing this in leaves us open to a rogue keeper attack where they just substitute a malicious rewardCollector contract